core/src/test/java/org/elasticsearch/search/aggregations/pipeline/InternalSimpleValueTests.java

```diff
@@ -67,4 +68,32 @@ protected void assertFromXContent(InternalSimpleValue simpleValue, ParsedAggrega
             assertEquals(parsed.value(), Double.NaN, 0);
         }
     }
+
+    @Override
+    protected InternalSimpleValue mutateInstance(InternalSimpleValue instance) {
+        String name = instance.getName();
+        double value = instance.getValue();
+        DocValueFormat formatter = instance.formatter();
+        List<PipelineAggregator> pipelineAggregators = instance.pipelineAggregators();
+        Map<String, Object> metaData = instance.getMetaData();
+        switch (between(0, 2)) {
+        case 0:
+            name += randomAlphaOfLength(5);
+            break;
+        case 1:
+            value += between(1, 100);
+            break;
+        case 2:
+        default:
```

Comment:
same as in the other PR, I would rather throw UnsupportedOperationException in the default case